app/views/missing_check_in_mailer/missing_check_in.html.erb

```diff
@@ -5,7 +5,7 @@
   </head>
   <body>
     <p>
-      Psssstt! Remember to <%= link_to "check-in for #{@friendly_date}", new_check_in_url(autofill_date: @friendly_date) %>
+      Psssstt! Remember to <%= link_to "check-in for #{@friendly_date}", new_check_in_url(check_in: { worked_at: @friendly_date }) %>
```

Comment:
And here we're passing the `check_in` into the `new_check_in_url` to set the worked at date to the `friendly_date` so that it can auto populate when the link is clicked.